common/localize.ts

```diff
@@ -79,6 +79,10 @@ const LOCALIZE_ID_TO_NAME = {
     he: 'נסה שוב',
   },
   //  TICTACTOE starts.
+  CONNECT4_GAME_NAME: {
```

Comment:
put your texts together inside 
// Connect4 starts.
...
// Connect4 ends

at the bottom